deps/build.jl

```diff
@@ -0,0 +1,2 @@
+using Blink
+Blink.AtomShell.isinstalled() || Blink.AtomShell.install()
```

Comment:
Why is this not part of Blink's build script?

Reply:
Yeah, I wish it were.